public/styles.css

```diff
@@ -1,14 +1,34 @@
+body {
+  background-color: #E5E5E5;
+}
+
+.header {
+  position: absolute;
+  height: 59px;
+  width: 100%;
+  left: -1px;
```

Comment:
We should be able to achieve this without the top and left values here. What problem are these values solving?

Reply:
The left value can be removed. Removing the top results in the following
![image](https://user-images.githubusercontent.com/70562257/110687323-3ab54300-8195-11eb-966d-1e9da5153446.png)